recipe/meta.yaml

```diff
@@ -1,50 +1,76 @@
-{% set version = "20170520" %}
+{% set version = "20180414" %}
 
 package:
   name: texlive-core
   version: {{ version }}
 
 source:
-  url: https://www.ctan.org/tex-archive/systems/texlive/Source/texlive-{{ version }}-source.tar.xz
-  sha256: c0a032aa9fa29e7cb521659779d6415f3b4b062a6a44193fb4e79cfefe7b8954
+  - url: https://www.ctan.org/tex-archive/systems/texlive/Source/texlive-{{ version }}-source.tar.xz
+    sha256: fe0036d5f66708ad973cdc4e413c0bb9ee2385224481f7b0fb229700a0891e4e
+    patches:
+      - texlive-20180414-source-upstream_fixes-1.patch
+  # 2.5GB download ..
```

Comment:
errr.  I dont thin that'll work

Reply:
Nope. It does not :smile:

Reply:
Probably. I don't have to time to investigate that now though, sorry.